site/layouts/partials/footer.html

```diff
@@ -12,16 +12,40 @@
                     {{ end }}
 
                     {{ if .GitInfo }}
-                    <strong>&middot;</strong>
-                    Built from commit
-                    <a href="https://github.com/kubernetes-sigs/krew/commit/{{- .GitInfo.Hash -}}"
-                        target="_blank" class="text-muted">
-                        {{- .GitInfo.AbbreviatedHash -}}
-                    </a>.
+                        &middot;
+                        <span id="lastmod"></span>
                     {{ end}}
                 </p>
-        </footer>
+            </footer>
         </div>
     </div><!-- .container -->
+
+    {{ if .GitInfo }}
+    <script>
+        var lastMod = {{- .Lastmod.Unix -}};
+        var commitHash = '{{- .GitInfo.Hash -}}';
+        function writeLastMod(){
```

Comment:
I find that `window.lastMod` rather ugly. After all, this sets up global variables. What about passing those two values to the function, like so:
```js
function writeLastMod(lastMod, commitHash) { ... }
```
Then you could also move the function into a static resource script which is included in the header. WDYT?

Reply:
I think we can keep the script here. It doesn’t make a lot of difference. If anything we don’t worry about old versions being cached (and hence incompatible with the new html) this way. 

But you are right. I can add the template var during invocation.

Reply:
Yeah, extracting the script is not critical. Either way is fine.